src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/CaseServiceImpl.java

```diff
@@ -1045,14 +1043,9 @@ private String mapToType(Reason reason) throws CTPException {
             rmCases
                 .stream()
                 .filter(c -> !(c.getCaseType().equals(CaseType.HI.name())))
```

Comment:
Filtering out HI cases after potentially filtering case events for these cases potentially wastes cpu and increases latency. There will potentially be many HI cases, not many for each household, but many all told.

Reply:
It's a good point, however I my guess is that the performance cost will be so small it will be impossible to measure (and dwarfed by other costs such as REST call costs).  Perhaps when we performance test CCSvc we can make sure the HI cases are representative to highlight any problems.